package com.zettamine.login.exception;

public class UserAlreadyExistsExcetpion extends RuntimeException {

	/**
	 * 
	 */
	private static final long serialVersionUID = 1L;

	public UserAlreadyExistsExcetpion() {
		super();
		// TODO Auto-generated constructor stub
	}

	public UserAlreadyExistsExcetpion(String message, Throwable cause, boolean enableSuppression,
			boolean writableStackTrace) {
		super(message, cause, enableSuppression, writableStackTrace);
		// TODO Auto-generated constructor stub
	}

	public UserAlreadyExistsExcetpion(String message, Throwable cause) {
		super(message, cause);
		// TODO Auto-generated constructor stub
	}

	public UserAlreadyExistsExcetpion(String message) {
		super(message);
		// TODO Auto-generated constructor stub
	}

	public UserAlreadyExistsExcetpion(Throwable cause) {
		super(cause);
		// TODO Auto-generated constructor stub
	}
	
	
	

}
